src/plugins/guided_onboarding/public/services/helpers.ts

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { guidesConfig } from '../constants/guides_config';
+import { GuideConfig, StepConfig, UseCase } from '../types';
+
+const getGuideConfig = (guideID?: string): GuideConfig | undefined => {
```

Comment:
[nit] We might want to export this. I think we use similar logic in the component.